libcontainer/integration/exec_test.go

```diff
@@ -1660,6 +1661,70 @@ func TestPIDHost(t *testing.T) {
 	}
 }
 
+func TestPIDHostInitProcessWait(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+
+	rootfs, err := newRootfs()
+	ok(t, err)
+	defer remove(rootfs)
+
+	pidns := "/proc/1/ns/pid"
+
+	config := newTemplateConfig(rootfs)
+	config.Namespaces.Add(configs.NEWPID, pidns)
+	container, err := newContainerWithName("test", config)
+	ok(t, err)
+	defer func() {
+		err := container.Destroy()
+		ok(t, err)
```

Comment:
You can't make any checks in defer, as the results of those will be discarded.

So, I guess, just
```go
defer container.Destroy()
```

Reply:
Using defer container.Destroy() caused the go linter to fail saying the return value of Destroy() was ignored. Hence I added the check. Should that linter error be ignored?